handbook/engineering/tracking_issues.md

```diff
@@ -40,6 +40,10 @@ Issues and pull requests from private repositories will have their title replace
 
 The generated work load can be re-ordered manually in the GitHub UI and that ordering will be preserved when the tracking issue is updated.
 
+#### Nesting
+
+Tracking issues can be nested within one another. This is helpful when tracking the implementation work of an RFC or a multiple-milestone goal.
```

Comment:
Anything in particular left to add here?

Reply:
What does it mean that they can nest?

Reply:
I'll add an example.

Reply:
Yep, maybe a screenshot.

Reply:
Also, we should make sure the other changes you're working on are documented, so that everyone can benefit from them :)

Reply:
Updated. I think this is the only _behavioral_ change. Everything else was visual, I think.